README.md

```diff
@@ -1,29 +1,8 @@
-# fork-me
-Yeah, right. FORK ME!!!
+#Hi, this is here Faisal irfan a web dev enthusiast
 
```

Comment:
This file shouldn't be changed

Reply:
Hi, sir i have undone the mistake 
The main Reademe.md file 

![10](https://github.com/atapas/fork-me/assets/97839373/a08b1d41-b428-4a45-bc22-27b732ff5a8b)